screens/HomeScreen.js

```diff
@@ -53,6 +59,12 @@ const HomeScreen = (props) => {
     }
   }, [props.profilePic]);
 
+  useEffect(() => {
+    getCalExpend();
+    getCalIntake();
+    getCalGoal();
+  })
```

Comment:
This use effect is unoptimized. Right now, the useEffect runs EVERYTIME an update happens, which can be taxing to the system by causing muliple re-renders if something changes in homescreen. I change profile picture, BOOM, use effect runs again. I click log out, state changes, BOOM, use effect runs again. What we want is to update once user is retrieved.

You fix that is by adding an array of values as the second argument of useEffect such that we don't re-render page unneccesarily. add [props.user] as second argument of this useEffect. You should be good!